compiler/src/dotty/tools/dotc/interpreter/RawExpr.scala

```diff
@@ -0,0 +1,7 @@
+package dotty.tools.dotc.interpreter
+
+import dotty.tools.dotc.ast.tpd
+
+class RawExpr(val tree: tpd.Tree) extends quoted.Expr[Any] with RawQuoted {
```

Comment:
All top-level classes merit a doc comment. Also, it would be good to have somewhere a diagram that shows the complete hierarchy rooted in `Quoted`.